tests/install/openqa_webui.pm

```diff
@@ -8,22 +8,41 @@ sub install_from_repos {
     diag('following https://github.com/os-autoinst/openQA/blob/master/docs/Installing.asciidoc');
     my $add_repo;
     if (get_required_var('VERSION') =~ /(tw|Tumbleweed)/) {
-        $add_repo = <<'EOF';
+        if (check_var('ARCH', 'ppc64le')) {
+            $add_repo = <<'EOF';
+zypper --non-interactive ar -f obs://devel:openQA/openSUSE_Factory_PowerPC openQA
+EOF
+        }
+        else {
+            $add_repo = <<'EOF';
 zypper --non-interactive ar -f obs://devel:openQA/openSUSE_Tumbleweed openQA
 EOF
+        }
     }
     elsif (check_var('VERSION', '42.1')) {
         $add_repo = <<'EOF';
 zypper ar -f obs://devel:openQA/openSUSE_Leap_42.1 openQA
 zypper ar -f obs://devel:openQA:Leap:42.1/openSUSE_Leap_42.1 openQA-perl-modules
 EOF
     }
+    elsif (check_var('VERSION', '42.2')) {
+        $add_repo = <<'EOF';
+zypper ar -f obs://devel:openQA/openSUSE_Leap_42.2 openQA
+zypper ar -f obs://devel:openQA:Leap:42.2/openSUSE_Leap_42.2 openQA-perl-modules
+EOF
+    }
     elsif (check_var('VERSION', 'SLES-12SP1')) {
         $add_repo = <<'EOF';
 zypper ar -f http://download.opensuse.org/repositories/devel:/openQA/SLE_12_SP1/devel:openQA.repo
 zypper ar -f http://download.opensuse.org/repositories/devel:/openQA:/SLE-12/SLE_12_SP1/devel:openQA:SLE-12.repo
 EOF
     }
+    elsif (check_var('VERSION', 'SLES-12SP2')) {
+        $add_repo = <<'EOF';
+zypper ar -f http://download.opensuse.org/repositories/devel:/openQA/SLE_12_SP2/devel:openQA.repo
+zypper ar -f http://download.opensuse.org/repositories/devel:/openQA:/SLE-12/SLE_12_SP2/devel:openQA:SLE-12.repo
+EOF
+    }
```

Comment:
these ones for Leap and SLES could be simply set by expanding the version in the URL and should not be hard-coded. I suggest to remove these changes for now. The power PC ones are fine